lib/appoptics_apm/inst/memcached.rb

```diff
@@ -58,15 +56,13 @@ def get_multi_with_appoptics(keys, raw = false)
           layer_kvs[:KVOp] = :get_multi
 
           AppOpticsAPM::API.trace(:memcache, layer_kvs || {}, :get_multi) do
-            info_kvs = {}
-            info_kvs[:KVKeyCount] = keys.flatten.length
+            layer_kvs[:KVKeyCount] = keys.flatten.length
 
             values = get_multi_without_appoptics(keys, raw)
 
-            info_kvs[:KVHitCount] = values.length
-            info_kvs[:Backtrace] = AppOpticsAPM::API.backtrace if AppOpticsAPM::Config[:memcached][:collect_backtraces]
+            layer_kvs[:KVHitCount] = values.length
```

Comment:
what is the goal in not sending an info message? are they deprecated?

Reply:
I think sending 2 instead of 3 events will be a performance improvement.
It also makes the trace data a bit easier to read, because there is always some repetition in the events of a span.